src/lib/modules/Document/DocumentInfo.js

```diff
@@ -58,6 +72,10 @@ export class DocumentInfo extends Component {
               <Table.Cell width={4}>Title</Table.Cell>
               <Table.Cell>{metadata.title}</Table.Cell>
             </Table.Row>
+            <Table.Row>
+              <Table.Cell width={4}>Alternative titles</Table.Cell>
+              {this.renderAlternativeTitles(metadata.alternative_titles)}
```

Comment:
I guess `alternative_titles` can be undefined so I would do something like `this.renderAlternativeTitles(_get('metadata.alternative_titles', []))` or `this.renderAlternativeTitles(metadata.alternative_titles || [])`  and remove the `isValidArray` check. That way we always expect an array or error otherwise.